spyder/app/tests/test_mainwindow.py

```diff
@@ -806,6 +806,11 @@ def test_runconfig_workdir(main_window, qtbot, tmpdir):
 @pytest.mark.skipif(sys.platform == "darwin", reason="Fails sometimes on Mac")
 def test_dedicated_consoles(main_window, qtbot):
```

Comment:
Please extract these changes to another PR so we can backport it to 6.x. This test started to fail too much, so we need to fix it in 6.x too in case we need to release an emergency version after 6.0.6.

Reply:
Created PR https://github.com/spyder-ide/spyder/pull/24399 with only the test changes

Reply:
Thanks @dalthviz! Then, please revert these changes here so they are only left in that other PR.